app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -90,6 +92,13 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
         })
     }
 
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
```

Comment:
This is ok for now, but I'll move this to our navigation drawer after merging this [PR](https://github.com/RocketChat/Rocket.Chat.Android/pull/725)

Reply:
:+1: